ui/widgets/playercontrols.go

```diff
@@ -139,6 +145,20 @@ func (pc *PlayerControls) SetPlaying(playing bool) {
 	}
 }
 
+func (pc *PlayerControls) OnChangeLoopMode(f func()) {
+	pc.loop.OnTapped = f
+}
+
+func (pc *PlayerControls) SetLoopMode(mode string) {
+	if mode == "all" {
+		themedResReplay.ColorName = theme.ColorNameSuccess
+		pc.loop.SetIcon(themedResReplay)
```

Comment:
The other case where I have a toggle button (the album filters button) I change the button color rather than the icon color to indicate it's active. I'm fine with either, but they should be consistent.

Reply:
I will go with the same approach you have for the album filters, to maintain consistency. :)
Later on, we can check whether changing the icon color looks better, but ideally that would require thinking not only about dark/light themes, but also planned support for custom themes.